app/protected/route.js

```diff
@@ -3,13 +3,12 @@ import Ember from 'ember';
 export default Ember.Route.extend({
 
   model(){
-
       return Ember.RSVP.hash({
         jobs: this.store.query('job', {
-          orderBy: "isActive",
-          equalTo: true
+          orderBy: "assignedTo",
```

Comment:
@donaldnash1989 ember fire doesn't have support for multiple filters in a query so I filter jobs by isActive in the protected [template.hbs](https://github.com/fdlane/fieldwork-worker/pull/6/commits/9ed298937a64a84c19a1a34f9f7da46abcf0b676#diff-604db37b3e632a05c5a0d8425deadeafR32).